index.md

```diff
@@ -1,8 +1,10 @@
 ---
 permalink: /
-title: 18F Partnership Playbook
+title: 18F Delivery: Partnership Playbook
```

Comment:
This needs to match exactly the `navigation:` entry in `_config.yml`. Running `./go update_nav` should do the trick.